examples/tutorial/tutorial_part_two.ipynb

```diff
@@ -232,7 +260,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "If you check out the accuracy of your frontend application again in the notebook from part 1, you'll see that it should be around 80% accuracy, about halfway between 74% and 88% accuracy."
+    "If you check out the accuracy of your frontend application again in the notebook from part 1, you'll see that it should be around 78-80% accuracy, about halfway between the accuracies of your SKLearn (68-74%) and TensorFlow (88%) accuracy models."
```

Comment:
Typo: s/the accuracies of your SKLearn (68-74%) and TensorFlow (88%) accuracy models/the accuracies of your SKLearn (68-74%) and TensorFlow (88%) models